build.sbt

```diff
@@ -124,33 +83,29 @@ lazy val `netty-ssl` = project
 
 lazy val server = project
   .in(file("modules/server"))
-  .dependsOn(common % "compile->compile;test->test")
-  .dependsOn(`internal-core` % "compile->compile;test->test")
-  .dependsOn(`internal-monix` % "test->test")
-  .dependsOn(`internal-fs2` % "test->test")
-  .dependsOn(channel % "test->test")
-  .dependsOn(testing % "test->test")
+  .dependsOn(`rpc-service`)
   .settings(moduleName := "mu-rpc-server")
   .settings(serverSettings)
 
 /////////////////////
 //// HEALTHCHECK ////
 /////////////////////
 
-lazy val `health-check-unary` = project
-  .in(file("modules/health-check-unary"))
-  .dependsOn(channel)
+lazy val `health-check` = project
+  .in(file("modules/health-check"))
+  .dependsOn(`rpc-service`)
+  .dependsOn(fs2)
+  .dependsOn(monix)
```

Comment:
> Combine the mu-health-check-unary, mu-rpc-fs2 and mu-rpc-monix modules into a single module called mu-health-check. This will contain both FS2 and Monix implementations, but it will declare dependencies with optional scope, so users only need to add a dependency on the streaming library they need. The grey arrows in the diagram indicate optional scope.

The original idea ☝️ . However, as we need to depend on fs2 and monix modules, I'm not sure how to _mark_ these module dependencies as `Optional`.

Reply:
I guess ideally with something like the [`provided` scope](https://howtodoinjava.com/maven/maven-dependency-scopes/#provided) in maven

Reply:
@cb372 's https://github.com/higherkindness/mu-scala/pull/900/commits/57f47cbb0ae1234b1f7903a57ac6cdebbb85f015 would also work fine :)